docs/openapi.json

```diff
@@ -1111,27 +1118,37 @@
         ],
         "responses": {
           "200": {
-            "description": "User was found. Returns all stored user information.",
+            "description": "Users matching criteria were found. Returns all stored user information.",
             "examples": {
-              "application/json": {
-                "email": "user@reana.info",
-                "id": "00000000-0000-0000-0000-000000000000",
-                "token": "Drmhze6EPcv0fN_81Bj-nA"
-              }
+              "application/json": [
+                {
+                  "email": "user@reana.info",
+                  "id": "00000000-0000-0000-0000-000000000000",
+                  "token": "Drmhze6EPcv0fN_81Bj-nA"
+                },
+                {
+                  "email": "user2@reana.info",
+                  "id": "00000000-0000-0000-0000-000000000001",
+                  "token": "Drmhze6EPcv0fN_81Bj-nB"
+                }
+              ]
             },
             "schema": {
-              "properties": {
-                "email": {
-                  "type": "string"
-                },
-                "id_": {
-                  "type": "string"
+              "items": {
+                "properties": {
+                  "email": {
+                    "type": "string"
+                  },
+                  "id_": {
+                    "type": "string"
+                  },
+                  "token": {
```

Comment:
I left the ACCESS_TOKEN as token here, to be included in the global renaming with https://github.com/reanahub/reana-client/issues/128